### PR TITLE
Ruff fixes

### PR DESCRIPTION
**Pull request by macaab26**
## What's changing

This PR is an attempt to "clean" the repo from the errors/suggestion that ruff creates prior to configure it in the action (that PR will come later!)

## Additional notes for reviewers

Ruff fixes!

## I already...

- [ ] added some tests for any new functionality;
- [ ] updated the documentation.

### CODE REVIEW

File: notebooks/lumigator_demo.py [hunk 1]
```diff
@@ -241,7 +241,7 @@ def parse_model_results(results):
 
             row[column] = value
 
-        row["RAM_GB"] = mi[mi.model_name == model_name]['RAM_GB'].values[0]
+        row["RAM_GB"] = mi[mi.model_name == model_name]['RAM_GB'].to_numpy()[0]
```
Comment (veekaybee) [inline notebooks/lumigator_demo.py:244]: we'll want to check this one just in case

**Comment by veekaybee** [comment]
LGTM pending the numpy check, I also checked the rules that we disabled/re-enabled for the lm-buddy import https://github.com/mozilla-ai/lumigator/issues/225

and those look ok as well, cc @aittalam

**Review by veekaybee — APPROVED** [review]
Pre-approving once we double-check a few items

File: notebooks/lumigator_demo.py (hunk 1 above)
Comment (aittalam) [reply notebooks/lumigator_demo.py:244]: Thanks @veekaybee ! Yes, it makes sense and I tested it, I confirm it works as expected 👍

**Review by aittalam — APPROVED** [review]
Many thanks @macaab26 , LGTM!

**Pull request merged**